### ISSUE TITLE
Error in IE11 browser

### ISSUE THREAD

**Issue by fidlip** [Daegalus/dart-uuid#37]
Given DartAngular web application with this simple code in web/main.dart
```
//import 'package:angular/angular.dart';
import 'package:dart_uuid_error_ie/app_component.template.dart' as ng;

import 'package:uuid/uuid.dart';

void main() {
  print(Uuid().v1());

 // runApp(ng.AppComponentNgFactory);
}

```
(actually it doesn't render anything into browser, only uses console)

Compile project with --release flag:
`webdev serve --release`
so it is compiled by _dart2js_ instead of _dartdevc_

- When tried in Chrome or Firefox it prints "6a1ef450-3260-11e9-99d8-e59f090dd440" to browsers console.
- When tried in InternetExplorer11 browser I get following error:
`No source of cryptographically secure random numbers available.`

library version: 2.0.0
dart-sdk version: 2.1.0
browser: IE 11

**Comment by Daegalus** [comment]
I'll look into this. There seems to be a dart sdk bug for secure random on ie11 and need to make sure it's not affecting this and how to workaround.

If you don't need secure random numbers you can pass the mathRNG function as the rng source. Which I thought I set as the default. I'll look over the code and make sure I didn't introduce any regressions.

Thanks for the detailed bug report.

I will try to get a fix out by tonight.

**Comment by fidlip** [comment]
Thanks a lot for response. I've tried to add that option, but no luck.

**Comment by Daegalus** [comment]
Can you do me a favor and test out master on IE11 for me. I have it installed but everytime I try to get into Developer Tools, IE11 hangs for me. I will release to pub once i get a thumbs up.

**Comment by fidlip** [comment]
Thank you Daegalus, It is working with master in IE11. Have a nice day.

**Comment by Daegalus** [comment]
Released 2.0.1 yesterday fixing this for the base case. Still need to figure out if Crypto works and if not, fix my IE11 from crashing.
